src/Foundation/Features/Checkout/Checkout.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model CheckoutViewModel
+
+@using (Html.BeginForm("PlaceOrder", "Checkout", FormMethod.Post, new { @class = "row jsCheckoutForm", id = "jsCheckoutForm", novalidate = "novalidate" }))
+{
+    @Html.AntiForgeryToken()
+    <div class="col-md-8 col-xs-12">
+        <div class="card items">
+            <div class="card-body">
+                @Html.Partial("ShippingInformation", Model)
+                <br />
+                @Html.Partial("SingleAddress", Model)
+                <hr />
+                @Html.Partial("BillingInformation", Model)
```

Comment:
Billing Information should be updated 

- Billing Information should be its own section with a border to distinguish Billing Information
- If there is only one shipment add a new radio button to Use Shipping Address for Billing Address as the first option
- Separate the Coupon partial view and move to the top of the checkout before the shipping information